main.h

```diff
@@ -18,11 +18,11 @@
  *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA
  */
 
-#ifndef _MAIN_H
-#define _MAIN_H 1
```

Comment:
Aren't macros with a leading underscore reserved? But somehow I frequently encounter this (at least in older GTK based projects).